Build.fsx

```diff
@@ -18,27 +18,48 @@ let nuGetPackages = !! (nuGetOutputFolder @@ "*.nupkg" )
 let signKeyPath = FullName "Src/AutoFixture.snk"
 let solutionsToBuild = !! "Src/All.sln"
 
-type BuildVersionInfo = { assemblyVersion:string; fileVersion:string; infoVersion:string; nugetVersion:string }
-let calculateVersionFromGit buildNumber =
+type VNextInfo = { name:string; forkedCommitSha:string }
+// To disable vNext branch set the vNextInfo variable to None
+// Specify sha1 of the master branch commit the vNext branch was forked from
+let vNextBranchInfo:Option<VNextInfo> = Some { name="v4"; forkedCommitSha = "a177824f05e911c70aac1a459b8da5bd206cf410" }
+
```

Comment:
This means one would have to look inside the build script and modify the F# code? Is this the only way? (I'm just asking because I currently don't have that much of experience in build automation stuff.)